src/ExtraSessionTrait.php

```diff
@@ -58,13 +62,17 @@ public function iWaitForCssElementBeingVisible($element, $seconds)
      * @param string $element
      * @param int    $seconds
      *
-     * @return bool
+     * @throws \RuntimeException
      *
      * @Given /^I wait for "([^"]*)" element being invisible for (\d+) seconds$/
      */
     public function iWaitForCssElementBeingInvisible($element, $seconds)
     {
-        return $this->getSession()->wait($seconds * 1000, sprintf("$('%s').length == false", $element));
+        $result = $this->getSession()->wait($seconds * 1000, sprintf("$('%s').length == false", $element));
```

Comment:
No issue with `.length == false`? DId you assert both success and fail cases?

Reply:
When I tested the change in our Behat test, the invisible function worked just as expected. I added tests to validate that RuntimeExceptions will be thrown if the expectation isn't met.